torch_tvm/compiler.cpp

```diff
@@ -27,11 +27,11 @@ tvm::relay::Var TVMCompiler::convertToRelay(Value* val) {
   AT_ASSERT(0);
 }
 
-tvm::relay::Expr TVMCompiler::convertToRelay(const IValue& val) {
+tvm::relay::Expr TVMCompiler::convertToRelay(const IValue& val, TVMContext ctx) {
   // All doubles are converted to floats
   if (val.isDouble()) {
     auto x = tvm::runtime::NDArray::Empty(
-        {}, tvm::runtime::String2TVMType("float32"), ctx_);
```

Comment:
I am a bit surprised this could access the`ctx_` before..

Reply:
it's a trick! this wasn't a static method before